data/AtmegaBuzz.json

```diff
@@ -0,0 +1,35 @@
+ "name": "Swapnil Shinde",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "println!() pro rust developer",
+  "avatar": "https://github.com/AtmegaBuzz.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/a_kraken_head",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://twitter.com/EddieAndSaraExplore",
```

Comment:
Please update this to use your own instagram link

Reply:
Hey @EmmaDawsonDev  Thankyou for pointing out. I fixed it can you review it now.